CenterForOpenScience/ember-osf-web
# [ENG-2095] Fix broken blur

- Ticket: [ENG-2095]
- Feature flag: n/a

## Purpose
- Fix blur logic introduced earlier

## Summary of Changes
- Remove global blur rule to `[inert]`
- Add blur rule to places `inert` css property was used (subject search/load, discover search result reloading)
- Remove `max-width` on dropdowns cause it looks funny in the draft registration metadata page

## Side Effects



## QA Notes
- the blur logic has been updated, so the subject search blur and the discover page results blur should stay the same. This is meant to address an issue that was introduced earlier that put a blur over the modal when claiming an unregistered contributor

[ENG-2095]: https://openscience.atlassian.net/browse/ENG-2095
